### ISSUE TITLE
Why the accuracy for validation set is zero?

### ISSUE THREAD

**Issue by marsggbo** [dragen1860/DARTS-PyTorch#3]
![image](https://user-images.githubusercontent.com/13477956/54476235-d1182d80-4835-11e9-9989-c8b12777e7f9.png)
